File: core/src/org/rexcellentgames/burningknight/entity/item/accessory/equippable/Scissors.java
package org.rexcellentgames.burningknight.entity.item.accessory.equippable;

public class Scissors extends Equippable {
	@Override
	public void onEquip(boolean load) {
		super.onEquip(load);
		this.owner.cutCobweb = true;
	}

	@Override
	public void onUnequip(boolean load) {
		super.onUnequip(load);
		this.owner.cutCobweb = false;
	}
}

File: core/src/org/rexcellentgames/burningknight/entity/item/accessory/equippable/Scissors.kt
package org.rexcellentgames.burningknight.entity.item.accessory.equippable

class Scissors : Equippable() {
	override fun onEquip(load: Boolean) {
		super.onEquip(load)
		this.owner.cutCobweb = true
	}

	override fun onUnequip(load: Boolean) {
		super.onUnequip(load)
		this.owner.cutCobweb = false
	}

	override fun canBeUpgraded(): Boolean {
		return false
	}

	override fun canBeDegraded(): Boolean {
		return false
	}
}